src/renderer/families/cosmos/DelegationFlowModal/fields/ValidatorField.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+import React, { useState } from "react";
+import type { TFunction } from "react-i18next";
+
+import { getAccountUnit } from "@ledgerhq/live-common/lib/account";
+import { useLedgerFirstShuffledValidatorsCosmos } from "@ledgerhq/live-common/lib/families/cosmos/react";
+
+import styled from "styled-components";
+import Box from "~/renderer/components/Box";
+import Text from "~/renderer/components/Text";
+import ScrollLoadingList from "~/renderer/components/ScrollLoadingList";
+import { Trans } from "react-i18next";
+import IconAngleDown from "~/renderer/icons/AngleDown";
+import ValidatorRow from "../components/ValidatorRow";
+import type { ThemedComponent } from "~/renderer/styles/StyleProvider";
+import type { Account, TransactionStatus } from "@ledgerhq/live-common/lib/types";
+import type {
+  CosmosDelegation,
+  CosmosValidatorItem,
+} from "@ledgerhq/live-common/lib/families/cosmos/types";
+
+type Props = {
+  t: TFunction,
+  account: Account,
+  status: TransactionStatus,
+  delegations: CosmosDelegation[],
+  onChangeValidator: ({ address: string }) => void,
+  chosenVoteAccAddr: string,
+};
+
+const ValidatorField = ({
+  account,
+  status,
+  t,
+  delegations,
+  onChangeValidator,
+  chosenVoteAccAddr,
+}: Props) => {
+  const [showAll, setShowAll] = useState(false);
+  const unit = getAccountUnit(account);
+  const validators = useLedgerFirstShuffledValidatorsCosmos();
+
+  const renderItem = (validator: CosmosValidatorItem, validatorIdx: number) => {
+    return (
+      <ValidatorRow
+        currency={account.currency}
+        key={validator.validatorAddress}
+        validator={validator}
+        unit={unit}
+        active={chosenVoteAccAddr === validator.validatorAddress}
+        onClick={onChangeValidator}
+      ></ValidatorRow>
+    );
+  };
+
+  return (
+    <ValidatorsFieldContainer>
+      <Box p={1}>
+        <ScrollLoadingList
+          data={
+            showAll
+              ? validators
+              : [validators.find(v => v.validatorAddress === chosenVoteAccAddr) || validators[0]]
+          }
+          style={{ flex: showAll ? "1 0 256px" : "1 0 64px", marginBottom: 0, paddingLeft: 0 }}
```

Comment:
Shouldn't define this style in its own `ThemedComponent`, same as others at the bottom?

Reply:
I'm not sure if we can do conditional